pulsar/provider.go

```diff
@@ -37,33 +37,36 @@ func Provider() terraform.ResourceProvider {
 		Schema: map[string]*schema.Schema{
 			"web_service_url": {
 				Type:        schema.TypeString,
-				Required:    true,
+				Optional:    true,
```

Comment:
why is this changed to `Optional`?

Reply:
`terraform-plugin-sdk` doesn't support the set `Required: true` in provider, so I added the default value for `web_service_url` in init the provider, when the `web_service_url` is empty, which will be set to `http://localhost:8080`.